Consul.Test/Consul.Test.csproj

```diff
@@ -30,7 +22,11 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="1.0.2" />
   </ItemGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1' OR '$(TargetFramework)' == 'netcoreapp3.1' ">
+  <ItemGroup Condition=" '$(TargetFramework)' != 'netcoreapp2.1' ">
+    <PackageReference Include="GitHubActionsTestLogger" Version="1.1.0" />
```

Comment:
`GitHubActionsTestLogger` works with 'netcoreapp2.1', just use version `1.2.0` and add the 
```
<CopyLocalLockFileAssemblies Condition="'$(TargetFramework)' == 'netcoreapp2.1'" >true</CopyLocalLockFileAssemblies>
```
to the test project as a workaround.